pyproject.toml

```
@@ -19,7 +19,7 @@ classifiers = [
     "Programming Language :: Python :: 3.9"
 ]
 dependencies = [
-    "numpy",
+    "numpy>=1.19.2,<1.20.0", # Modified numpy version
```

```suggestion
    "numpy>=1.19.2",
```

We don't want to prevent newer versions from being installed, we just want it to be compatible with older versions.

---

Done